Instructions/Labs/04-exercise-enrich-cognitive-custom-classes.md

```diff
@@ -0,0 +1,531 @@
+You've built a search solution and now want to add Azure AI Services for language enrichments to your indexes.
+
+:::image type="content" source="../media/04-media//exercise-overview.png" alt-text="A diagram showing the components of this exercise. Azure Cognitive Search, Azure AI Language, and a Function App." border="false":::
+
+In this exercise, you'll create an Azure Cognitive Search solution and enrich an index with the results from a Language Studio custom text classification project. You'll create a function app to connect search and your classification model together.
+
+> [!NOTE]
+> To complete this exercise, you will need a Microsoft Azure subscription. If you don't already have one, you can sign up for a free trial at [https://azure.com/free](https://azure.com/free?azure-portal=true).
+
+
+## Set up your development environment with Python, VS Code and VS Code Extensions
+
+Install these tools to complete this exercise. You can still follow along with the steps without these tools.
+
+1. Install [VS Code](https://code.visualstudio.com/)
+1. Install [Azure Core Functions Tool](https://github.com/Azure/azure-functions-core-tools)
+1. Install [Azure Tools Extensions for VS Code](https://code.visualstudio.com/docs/azure/extensions)
+1. Install [Python 3.8](https://www.python.org/downloads/release/python-380/) for your operating system.
+1. Install [Python Extension for VS Code](https://marketplace.visualstudio.com/items?itemName=ms-python.python)
+
+
+## Set up your Azure resources
+
+To save you time, select this Azure ARM template to create resources you'll need later in the exercise.
+
+### Deploy a pre-built ARM template
+
+1. [![Deploy to Azure.](../media/04-media//deploy-azure.svg)](https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FMicrosoftLearning%2Fmslearn-doc-intelligence%2Fmain%2Fcognitive-search%2Fazuredeploy.json) select this link to create your starting resources. You might need to copy and paste the [direct link](https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FMicrosoftLearning%2Fmslearn-doc-intelligence%2Fmain%2Fcognitive-search%2Fazuredeploy.json) into your search bar.
```

Comment:
extra `/`?

Reply:
Thanks for spotting. I used CTRL+F and got some rogue // in there :) Fixed!